tests/cli/profiles.yml

```diff
@@ -0,0 +1,11 @@
+jaffle_shop:
```

Comment:
The way these test files being stored is similar to the old testing framework. Can we test it similar to what the functional tests does?
cc @gshank